include/marl/pool.h

```diff
@@ -214,32 +212,65 @@ class BoundedPool : public Pool<T> {
 
   // borrow() borrows a single item from the pool, blocking until an item is
   // returned if the pool is empty.
-  MARL_NO_EXPORT inline Loan borrow() const;
+  template <typename... Args>
+  MARL_NO_EXPORT inline Loan borrow(Args&&... args) const {
```

Comment:
Any reason you've inlined these methods into the class? Tint tries to keep the implementations out of the class, so the public interface is easily human parsable. I'd prefer it if these were put back where they were.